Interfaces/API/Dirac.py

```diff
@@ -65,6 +66,25 @@ def parseArguments(args):
     argList += arg.split(',')
   return argList
 
+def fileCatalogAPIWrapper(func):
```

Comment:
that's more a debugging function for you, not really something to go to the API

Reply:
Well, this is a wrapper to hide away all the logging code which is identical for most of the methods. Otherwise a lot of code repetition would occur.

Reply:
Please, do add FC related stuff to Dirac.py. FileCatalog() is itself the API.

Reply:
OK, but there was some FC related stuff there already. I've just added more ;-). What should go and what should stay in Dirac.py ?

Reply:
The only pure FC method exposed through Dirac.py is getAllReplicas() which I think was a mistake to introduce. All the others do something on top of the FC queries. So, I do not think you should add anything at all to Dirac.py

Reply:
Please drop them @martynia

Reply:
I understand you don't want to pollute Dirac.py with FC stuff, but it would be very useful to manipulate  user metadata programmatically (and there is server code for that). So is there any suitable place to move the client APi I have written ?

Reply:
And the tests would be useless since they use the client methods.